ZipBuilder/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -223,7 +227,8 @@ struct FrameworkBuilder {
                            "ARCHS=\(arch.rawValue)",
                            "BUILD_DIR=\(buildDir.path)",
                            "-sdk", platform.rawValue]
-    let args = standardOptions + platform.extraArguments()
+    let distributionFlag = carthageBuild ? "-DFIREBASE_BUILD_CARTHAGE" : "-DFIREBASE_BUILD_ZIP_FILE"
```

Comment:
Perhaps add Swift PM support in now too?

Reply:
Discussed offline, this is purely for binary distribution and SwiftPM support will look different in the future, likely. Will revisit at a later date once we have a better idea of what SwiftPM support and distribution looks like!